src/main/java/org/prebid/server/bidder/sharethrough/model/Size.java

```diff
@@ -0,0 +1,12 @@
+package org.prebid.server.bidder.sharethrough.model;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Builder
+@Value
+public class Size {
+    int height;
```

Comment:
Here and in all other models - please change the field types from primitives to their wrappers. Using primitives requires us to have a value, while in some cases it might be missing or unavailable.